tests/observability/main_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package observability
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/resource/environment"
+)
+
+var (
+	inst istio.Instance
+)
+
+func TestMain(m *testing.M) {
+	framework.
+		NewSuite("observability", m).
+		SetupOnEnv(environment.Kube, istio.Setup(&inst, setupConfig)).
+		RequireEnvironment(environment.Kube).
+		Run()
+}
+
+
+func setupConfig(cfg *istio.Config) {
+	if cfg == nil {
+		return
+	}
+	cfg.ControlPlaneValues = `
+values:
+  global:
+    proxy:
+      accessLogFile: "/dev/stdout"
+`
+}
```

Comment:
Instead of configuring this in Go code, it might be better to just start the test without access logging enabled, and then you could test the doc more thoroughly by calling `snip_enable_envoys_access_logging_1` and `snip_disable_envoys_access_logging_1` in your test.

Reply:
@frankbu `snip_enable_envoys_access_logging_1 ` can not be run as no manifest bundled in dev build. So I changed to using go code.

Reply:
@irisdingbj I'm not sure I understand. Are you saying that running this command:
```
istioctl manifest apply --set profile=demo --set meshConfig.accessLogFile="/dev/stdout"
```
will fail?

If so, we need to fix the test env to allow it to run, since this is needed for all the install tests in the `Setup` docs.

For this test, I think we don't even need to install Istio at all before running the script. See my other suggested change, to remove the call to `SetupOnEnv`.